scripts/grep_pv

```diff
@@ -16,21 +16,24 @@ fi
 KEYWORD=$1
 
 if [[ -n $KEYWORD ]]; then
+    # shellcheck disable=SC2164
     cd /reg/d/iocData/
-    for i in ioc-*; do 
-	cd /reg/d/iocData/$i
+    for i in ioc-*; do
+	# shellcheck disable=SC2164
+	cd /reg/d/iocData/"$i"
 	if [ -e ./iocInfo/ ] ; then
+	    # shellcheck disable=SC2164
      	    cd iocInfo
 	    if [ -e ./IOC.pvlist ] ; then
```

Comment:
And even with these exist checks, it's not obvious to me what they gain for us compared to letting the grep fail.